.circleci/config.yml

```
@@ -83,23 +104,45 @@ jobs:
       - checkout
       # npm i or restore cache
       - <<: *restore_dependency_cache_win
-      # install selenium 
+      # install selenium
       - run: |
           choco install selenium-ie-driver --version 3.141.5
           export PATH=/c/tools/selenium:$PATH
           echo $PATH
       # build `axe`
       - run: npm run build
       # get fixtures ready for running tests
-      - run: npx grunt testconfig	
-      - run: npx grunt fixture	
+      - run: npx grunt testconfig
+      - run: npx grunt fixture
       # run IE webdriver tests
       - run: npx grunt connect test-webdriver:ie
       # test examples
       # Note: Jasmine karma-chrome-launcher requires chrome browser
       - run: choco install googlechrome --ignore-checksums
       - run: npm run test:examples
 
+  # Run test suite in safari in macOS
+  test_macos:
+      <<: *defaults
+      <<: *macos_box
+      steps:
+        - checkout
+        # npm i or restore cache
+        - <<: *restore_dependency_cache_macos
+        # build `axe`
+        - run: npm run build
+        # get fixtures ready for running tests
+        - run: npx grunt testconfig
+        - run: npx grunt fixture
+        # enable connection to safaridriver
+        - run: |
+            defaults write com.apple.Safari IncludeDevelopMenu YES
+            defaults write com.apple.Safari AllowRemoteAutomation 1
+            sudo safaridriver --enable
+            safaridriver -p 0 &
```

This should be a ["background" command](https://circleci.com/docs/2.0/configuration-reference/#background-commands).

```suggestion
        - run: |
            defaults write com.apple.Safari IncludeDevelopMenu YES
            defaults write com.apple.Safari AllowRemoteAutomation 1
            sudo safaridriver --enable
        - run:
              command: safaridriver -p 0
              background: true
```